ocdskingfisherprocess/transform/upgrade_1_0_to_1_1.py

```diff
@@ -0,0 +1,83 @@
+from ocdskingfisherprocess.transform.base import BaseTransform
+from ocdskit.upgrade import upgrade_10_11
+import sqlalchemy as sa
+
+
+class Upgrade10To11Transform(BaseTransform):
+    type = 'UPGRADE-1-0-TO-1-1'
+
+    def process(self):
+        self.process_all_files()
+
+    def process_all_files(self):
+        for file_model in self.database.get_all_files_in_collection(self.source_collection.database_id):
+            self.process_file(file_model)
+
+    def process_file(self, file_model):
+        for file_item_model in self.database.get_all_files_items_in_file(file_model):
+            self.process_file_item(file_model, file_item_model)
+
+    def process_file_item(self, file_model, file_item_model):
+        with self.database.get_engine().begin() as connection:
+            release_rows = connection.execute(
+                self.database.release_table.select().where(
+                    self.database.release_table.c.collection_file_item_id == file_item_model.database_id)
+            )
+
+        for release_row in release_rows:
+            if not self.has_release_id_been_done(release_row['id']):
+                self.process_release_row(file_model, file_item_model, release_row)
+
+        del release_rows
```

Comment:
Wouldn't it be clearer to use SQLAlchemy's `.close()` function?

Reply:
Whatever the right thing to do is, it should also be done to the checks code. So I'll make an issue to look into this, and do the right thing to both. Thanks.